kedro/templates/project/{{ cookiecutter.repo_name }}/setup.cfg

```diff
@@ -1,3 +1,7 @@
+[flake8]
+max-line-length = 88
+extend-ignore = E203
+
 [isort]
```

Comment:
Two more questions here:

1. Is there any reason to set `force_grid_wrap=0`? It is the default.
2. Do we need `known_third_party=kedro`? It should be detected like any other library.

Reply:
Hey @deepyaman!

Re 1: I think the reason was purely to comply with [Black suggestions](https://black.readthedocs.io/en/stable/compatible_configs.html). Otherwise things may break if isort (ever) decide to change its defaults.

Re 2: It might help with running some unit/e2e tests as Kedro might not be installed at that point but rather just be present in the current working dir (so might be considered a 1st party package, I think)